charts/jenkins/values-casc-jobs.yaml.gotmpl

```diff
@@ -80,6 +80,15 @@ controller:
         - script: > {{ readFile "jobs/nuxeo/lts/release-nuxeo-2021.groovy" | nindent 12 }}
         - script: > {{ readFile "jobs/nuxeo/lts/release-nuxeo-hf-2021.groovy" | nindent 12 }}
         - script: > {{ readFile "jobs/nuxeo/lts/release-nuxeo-jsf-ui-2021.groovy" | nindent 12 }}
+        - script: >
+            folder('nuxeo/lts/2023')
```

Comment:
I think we should move related jobs for 2021 to the 2021 folder, or put every 2023 jobs at the same level, or make each jobs lts version agnostic by adding a job parameter.

Reply:
I would vouch for the 2021 folder.

Reply:
So do I